test/db.go

```diff
@@ -40,7 +53,7 @@ func ResetIncidentsTestDatabase(t testing.TB) func() {
 }
 
 func resetTestDatabase(t testing.TB, ctx context.Context, dbPrefix string) func() {
-	db, err := sql.Open("mysql", fmt.Sprintf("test_setup@tcp(boulder-proxysql:6033)/%s_sa_test", dbPrefix))
+	db, err := sql.Open("mysql", fmt.Sprintf("test_setup@tcp(%s)/%s_sa_test", dbAddr, dbPrefix))
```

Comment:
Since this is the only use in this package, we could avoid the duplication of `dbAddr` by changing ResetBoulderTestDatabase and ResetIncidentsTestDatabase to reference DBConnSAFullPerms / DBConnIncidentsFullPerms from the `test/vars` package. Would require this function to take a DSN rather than a dbPrefix.